src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/JsonLiteral.java

```diff
@@ -4,10 +4,10 @@
 /**
  * @param value Literal value to use as RHS operand in the query
  * @param type Literal value type of the RHS operand in the query
- * @param <DataType> Data type of the value, the value should be the Java object value extracted
```

Comment:
Additional change to follow Java best practices: single Type parameter usually `T` and always upper-case.

Reply:
Agreed: I would not make similar change for multi-type case OR if the type variable was not a single contained/wrapped type.

But for a single "simple" type, `T` is pretty much standard.